Added Tests for get_page in api

DartingMelody opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## General Checkups
- [x ] Have you checked that there aren't other open pull requests for the same issue/update/change?
- [x ] If your code includes new features and not just bug fixes/copy editing: Did you include new tests?

<!---
by the way: if you want to discuss feature requests and bugs with us before submitting anything on GitHub: We're always happy to chat on Slack: http://slackin.openhumans.org/
 -->

## Description
<!--- Describe your changes in detail -->

## Related Issue
<!--- if this closes an issue make sure include e.g., "fix #4"
or similar - or if just relates to an issue make sure to mention
it like "#4" -->

## Example
<!--- if introducing a new feature or changing behavior of existing
methods/functions, include an example if possible to do in brief form -->

<!--- Did you remember to include tests? Unless you're just changing
grammar, please include new tests for your change -->

DartingMelody commented:
hi @madprime , Please review the PR. Thanks

madprime commented on ohapi/cassettes/test_get_page_with_results.yaml:
```diff
@@ -0,0 +1,27 @@
+      Connection: [keep-alive]
+      User-Agent: [python-requests/2.9.1]
+    method: GET
+    uri: https://www.openhumans.org/api/direct-sharing/project/exchange-member/?access_token=dQ5Phvwaz2kthlKnv0I4ytulMt8fBG
```
the access_token should be filtered out and replaced with a placeholder

madprime commented on ohapi/tests/test_get_page.py:
```diff
@@ -0,0 +1,33 @@
+                 filter_query_parameters=['access_token'])
+
+
+class test_get_page(unittest.TestCase):
```
I'd prefer for `test_api.py` to have all tests relevant to `api.py`. Could you please move this class to that file?

Also, I'd like the class to be renamed to conform to PEP8 (https://www.python.org/dev/peps/pep-0008/#class-names), e.g. `APITestGetPage`

DartingMelody commented:
Hi @gedankenstuecke @madprime I have renamed and moved the tests for get_page function. I have also  set filter query parameters as suggested. However, there is one hound violation which is occurring because of a locals() function call. Please review.

madprime commented:
@DartingMelody Thanks, looks good! No worries on Hound's issue due to use of locals(), that's expected. :)
